pnpm-workspaces/packages/two/BUILD.bazel

```diff
@@ -0,0 +1,33 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project", "ts_config")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+
+npm_link_all_packages(name = "node_modules")
+
+ts_config(
+    name = "tsconfig",
+    src = "tsconfig.json",
+    deps = ["//:tsconfig"],
+    visibility = ["//visibility:public"],
+)
+
+ts_project(
+    name = "twotwo",
+    srcs = ["src/main.ts"],
+    tsconfig = ":tsconfig",
+    visibility = ["//visibility:public"],
+
+    deps = [
+        "//:node_modules/@types/node",
+    ],
+)
+
+npm_package(
+    name = "two",
+    srcs = [
+        "src/main.js",
```

Comment:
Do you think it's clearer to do `src/main.js` or `:twotwo`?

I think if the `ts_project` had multiple files this would only put this one file into the package?

Reply:
Good point!

Reply:
I often do something such as `_ts` or maybe `_two_ts`.